packages/orbit-components/src/Tabs/components/Tab/types.d.ts

```diff
@@ -0,0 +1,10 @@
+import type { Globals } from "../../../common/types";
+
+export type Type = "default" | "basic" | "medium" | "top";
```

Comment:
Since we are defining the enum, we should take advantage of it and create the type from the enum. By doing that, we avoid possible inconsistencies in the future, as we only need to change new types in one place.